docs/sdk/serverless-signing/run-once.md

```diff
@@ -0,0 +1,61 @@
+import FeedbackComponent from "@site/src/pages/feedback.md";
+
+# Run on a Single Node Within an Action
+
+## Overview
+Witin a Lit action, you may want only a single node to perform a specific piece of your lit action logic, and let the result be broadcast to all other nodes which are executing the same Lit action in parallel. `runOnce` takes a function as a parameter, and all the nodes use a deterministic algorithm to choose which node will run the function that is passed to runOnce. This single node runs the function, and then broadcasts the result to all the other nodes.
```

Comment:
Copy suggestion:

Current:
"Witin a Lit action, you may want only a single node to perform a specific piece of your lit action logic, and let the result be broadcast to all other nodes which are executing the same Lit action in parallel. `runOnce` takes a function as a parameter, and all the nodes use a deterministic algorithm to choose which node will run the function that is passed to runOnce. This single node runs the function, and then broadcasts the result to all the other nodes."

Suggestion:
"Typically, when a Lit Action is called it is executed across every Lit node in parallel. With `runOnce`, you have the ability to perform specified operations on a _single_ node, versus all of them at once. 

The `runOnce` function takes another function(s) as a parameter and a deterministic algorithm is used to select the Lit node that it will be executed on. This node will run the function and broadcast the result to all of the other Lit nodes."